### ISSUE TITLE
DRM Decoder Plugin

### ISSUE THREAD

**Issue by var-IO-reeks** [AlexandreRouma/SDRPlusPlus#1341]
**Feature description**
I think a Digital Radio Mondiale (DRM) decoder plugin would be a nice addition to SDR++, since the current standalone DRM programs all have issues and aren't very user friendly.
Using them as a base for a plugin within the already feature-rich ecosystem of SDR++ would solve a bunch of their problems (limited source options, unstable GUI etc.).

Apart from the now pretty much abandoned DREAM DRM decoder, this projects looks for example like a possible base for such a plugin, since it is also written in C++:

https://github.com/JvanKatwijk/drm-receiver

I would love to try to port that by myself, but because of my extremely limited C++ knowledge, that will be a "difficult" task 😬

**Comment by Axpelle** [comment]
Agreed. But the suggestion for a built-in plugin is still worth considering. Third party DRM solutions tend to be flaky at best.

**Comment by AlexandreRouma** [comment]
> But the suggestion for a built-in plugin is still worth considering.

I would have closed the issue if it wasn't

**Comment by stressangrydisapoitment** [comment]
2 years later...any progress?
